src/core/server/ui_settings/settings/date_formats.ts

```diff
@@ -168,7 +168,9 @@ export const getDateFormatSettings = (): Record<string, UiSettingsParams> => {
         defaultMessage: 'Used for the {dateNanosLink} datatype of OpenSearch',
         values: {
           dateNanosLink:
-            '<a href="https://www.opensearch.org/guide/en/elasticsearch/reference/master/date_nanos.html" target="_blank" rel="noopener noreferrer">' +
+            // TODO: [RENAMEME] Need prod urls.
+            // issue: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/335#issuecomment-868294864
+            '<a href="https://docs-beta.opensearch.org/opensearch/units" target="_blank" rel="noopener noreferrer">' +
```

Comment:
Nice. learned 'Adding rel="noopener noreferrer" to links protects your site's users against having the site you've linked to potentially hijacking the browser'

Reply:
Created an issue for it.